dolphin/coordination.py

```diff
@@ -64,8 +77,21 @@ def start(self):
 
         # NOTE(gouthamr): Tooz expects member_id as a byte string.
         member_id = (self.prefix + self.agent_id).encode('ascii')
+        if cfg.CONF.coordination.backend_user \
```

Comment:
Please add some comments about this logic

Reply:
Done